README.md

```diff
@@ -89,7 +89,7 @@ Alternatively, comment on our issues if you plan to solve one.
 
 ## Analytics
 
-We track users to improve our application. We store your GitHub username and whether your Watermelon query was successful or not.
+We track user actions in an anonymous manner to improve our application. We store your GitHub username, repo name and organization.
```

Comment:
We don't store GH username anymore, right?

Reply:
We do https://vscode.dev/github/watermelontools/wm-extension/blob/c87ca0f4096f86b80ca9f1008fbc4785933a4cd7/src/extension.ts#L124